lightning/src/lib.rs

```diff
@@ -17,6 +17,40 @@
 //! figure out how best to make networking happen/timers fire/things get written to disk/keys get
 //! generated/etc. This makes it a good candidate for tight integration into an existing wallet
 //! instead of having a rather-separate lightning appendage to a wallet.
+//! 
+//! Some of the features include: 
+//! 
+//! 1. Internal test utilities exposed to other repo crates:
+//! ```rust
+//! _test_utils = ["hex", "regex", "bitcoin/bitcoinconsensus"]
+//! ```
+//! 
+//! 2. Unlog messages superior at targeted level:
+//! ```rust
+//! max_level_off = []
+//! max_level_error = []
+//! max_level_warn = []
+//! max_level_info = []
+//! max_level_debug = []
+//! max_level_trace = []
+//! ```
+//! 
+//! 3. Allow signing of local transactions that may have been revoked or will be revoked, for functional testing (e.g. justice tx handling). This is unsafe to use in production because it may result in the counterparty publishing taking our funds.
+//! ```rust
+//! unsafe_revoked_tx_signing = []
+//! _bench_unstable = []
+//! 
+//! no-std = ["hashbrown", "bitcoin/no-std", "core2/alloc"]
+//! std = ["bitcoin/std"]
+//! ```
+//! 
+//! 4. Generate low-r bitcoin signatures, which saves 1 byte in 50% of the cases:
```

Comment:
Mh, from/for what does it save 1 byte in 50%? Could you explain what the feature does and maybe supply a reference for further reading, e.g., https://github.com/bitcoin/bitcoin/pull/13666 or https://bitcoin.stackexchange.com/questions/111660/what-is-signature-grinding?

Reply:
@TheBlueMatt @tnull thank you for your comments and suggestions! I made the recommend changes!